label_studio_ml/examples/gliner/test_api.py

```diff
@@ -0,0 +1,47 @@
+"""
+This file contains tests for the API of your model. You can run these tests by installing test requirements:
+
+    ```bash
+    pip install -r requirements-test.txt
+    ```
+Then execute `pytest` in the directory of this file.
+
+- Change `NewModel` to the name of the class in your model.py file.
+- Change the `request` and `expected_response` variables to match the input and output of your model.
+"""
+
+import pytest
+import json
+from model import NewModel
```

Comment:
it should import the actual gliner class.
Ensure you can run tests successfully with `pytest` command inside this directory.